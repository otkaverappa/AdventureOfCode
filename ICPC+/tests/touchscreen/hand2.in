1
qwertyuiopasdfghjklzxcvbnm 3
qwertyuiopsdfghjklzxcvbnma
mnbvcxzlkjhgfdsapoiuytrewq
mnbvcxzlkjhgfdsapoiuytrewa
