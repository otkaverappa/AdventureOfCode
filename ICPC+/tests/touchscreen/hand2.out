qwertyuiopsdfghjklzxcvbnma 30
mnbvcxzlkjhgfdsapoiuytrewa 109
mnbvcxzlkjhgfdsapoiuytrewq 110
